template/base/_pnpm-workspace.yaml

```
@@ -1,4 +1,9 @@
 # managed by skuba
+minimumReleaseAge: 4320
```

Not really fussed, but thoughts on:

```suggestion
minimumReleaseAge: 4320 # 3 days
```

---

Like it saves people having to ask what it means 😆